arbnode/batch_poster.go

```diff
@@ -184,6 +204,24 @@ type BatchPosterConfig struct {
 
 	gasRefunder  common.Address
 	l1BlockBound l1BlockBound
+	// Espresso specific flags
+	EspressoTeeType                  string                                   `koanf:"espresso-tee-type"`
```

Comment:
I have an idea about this, lets make a big config called as EspressoConfig which takes in all these params. This way we will always have one config here and then we can keep adding stuff within that. Let me know, doesnt have to be in this PR!